src/common/node/childProcess.ts

```diff
@@ -77,7 +77,7 @@ export class ChildProcess {
     }
 
     public spawn(command: string, args?: string[], options: ISpawnOptions = {}): ISpawnResult {
-        let outcome = Q.defer<number>();
+        let outcome = Q.defer<void>();
```

Comment:
This promise is never resolved, only rejected. We might need to re-examine our workflow around here.